### DISCUSSION TITLE
How open a PDFtempfile

### DISCUSSION THREAD

**Discussion by zalmanchen** [pymupdf/PyMuPDF#2268] [Q&A]
![image](https://user-images.githubusercontent.com/70104951/221427363-98f0299e-16e7-4d68-8a72-07d7ec0fd96a.png)

My program needs to read tempfile, but fitz cannot read it

**Comment by JorjMcKie** [comment]
The file is empty. Documents cannot be created from empty files.

**Reply by zalmanchen** [reply]
Temporary files cannot be read, so I later saved them through _shutil.copy_, and they were fine.

**Comment by JorjMcKie** [comment]
This is no issue, but a "Discussions" item. Please follow the link to there.

**Reply by zalmanchen** [reply]
Sorry, I'll follow it later. Thank you for your suggestion.
